libs/sdk-ui-kit/src/Header/tests/generateHeaderHelpMenuItems.test.ts

```diff
@@ -0,0 +1,226 @@
+// (C) 2021 GoodData Corporation
+import { generateHeaderHelpMenuItems } from "../generateHeaderHelpMenuItems";
+import { getHelpMenuFeatureFlagsMock } from "./mock";
+
+describe("generateHeaderHelpMenuItems", () => {
+    it("should return documentation, portal and ticket item", () => {
+        const items = generateHeaderHelpMenuItems(
+            "https://test.gooddata.com/documentation",
+            "https://test.gooddata.com/support",
+            "test@gooddata.com",
+            "TestWorkspaceId",
+            "TestSessionId",
+            "supportEmail@gooddata.com",
+            true,
+            getHelpMenuFeatureFlagsMock(true, true),
+        );
+        expect(items).toEqual([
+            {
+                className: "s-documentation",
+                href: "https://test.gooddata.com/documentation",
+                key: "gs.header.documentation",
+                target: "_blank",
+            },
+            {
+                className: "s-support-portal",
+                href: "https://test.gooddata.com/support",
+                key: "gs.header.visitSupportPortal",
+                target: "_blank",
+            },
+            {
+                className: "s-submit-ticket",
+                href: "mailto:supportEmail@gooddata.com",
+                key: "gs.header.submitTicket",
+                target: "_blank",
+            },
+        ]);
+    });
+
+    it("should not return documentation item", () => {
```

Comment:
Incomplete test name, should probably be

> should not return documentation item if documentationUrl is not specified

Reply:
Many other tests in this suite have similar problem, please review them

Reply:
I have changed like suggestion above thanks